basetest.pm

```diff
@@ -367,6 +368,27 @@ sub next_resultname {
     }
 }
 
+=head2 record_resultfile
+
+    $self->record_resultfile($title, $output [, result => $result] [, resultname => $name]);
+
+Record result file to be parsed when evaluating test results, for example
+within the openQA web interface.
+=cut
+sub record_resultfile {
+    my ($self, $title, $output, %nargs) = @_;
+    my $filename = $self->next_resultname('txt', $nargs{resultname});
```

Comment:
AFAIK just forwarding an undef value does not produce a warning as the value is not actually "used" here. see https://github.com/os-autoinst/os-autoinst/pull/720 for my upcoming changes about preventing warnings to slip through within the execution of tests. btw, I think you can help with https://github.com/os-autoinst/os-autoinst/pull/720/commits/278c46cad82692bbee45fc0396d5cba1206a3367#diff-200ab83e6b455d66c0ea0a727f5f6dd1 :-)